Sources/Sentry/Profiling/SentryProfilerSerialization.mm

```diff
@@ -389,6 +385,10 @@
         return nil;
     }
 
+#    if defined(TEST) || defined(TESTCI) || defined(DEBUG)
```

Comment:
h: 
We should refrain from using `defined(DEBUG)` for testing things, this will also run from cocoapods

Reply:
Excellent catch; I just copy-pasted the code. Thanks 🥇